package.json

```diff
@@ -22,7 +22,7 @@
     "postinstall": "./scripts/postinstall.sh",
     "clean": "ocular-clean",
     "build": "npm run clean && ocular-build && lerna run build",
-    "version": "ocular-build core,carto",
+    "version": "npm run build",
```

Comment:
Unrelated nit: Isn't it a pity to run `clean` on every build as it defeats `tsc`s incremental build speed gains. Longer-term clean could be done as part of the publish script only?